package.json

```diff
@@ -11,7 +11,8 @@
     "react-dom": "^17.0.1",
     "regenerator-runtime": "^0.13.7",
     "typescript": "^4.2.4",
-    "webpack": "^5.33.2"
+    "webpack": "^5.33.2",
+    "yarn": "^1.22.10"
```

Comment:
@veljkocukic there's no need to have this file here since Yarn should be globally installed not locally (in the project).